src/web/utils/severity.js

```diff
@@ -49,13 +58,34 @@ export const severityRiskFactor = value => {
     return MEDIUM;
   }
 
-  if (isDefined(high) && value >= high) {
+  if (value >= high && isDefined(critical) && value < critical) {
     return HIGH;
   }
 
+  if (isDefined(critical) && value >= critical) {
+    return CRITICAL;
+  }
+
   return NA;
 };
 
+export const severityRiskFactorToValue = factor => {
```

Comment:
Question: what about map object lookup instead of switch statement? 
```suggestion
const severityMap = {
  [CRITICAL]: CRITICAL_VALUE,
  [HIGH]: HIGH_VALUE,
};

export const severityRiskFactorToValue = factor => severityMap[factor];
```